save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/controller/HeartbeatController.kt

```diff
@@ -71,7 +90,7 @@ class HeartbeatController(private val agentService: AgentService,
                         }
                     }
                     AgentState.BUSY -> Mono.just(ContinueResponse)
-                    AgentState.BACKEND_FAILURE, AgentState.BACKEND_UNREACHABLE, AgentState.CLI_FAILED, AgentState.STOPPED_BY_ORCH -> Mono.just(WaitResponse)
+                    AgentState.BACKEND_FAILURE, AgentState.BACKEND_UNREACHABLE, AgentState.CLI_FAILED, AgentState.STOPPED_BY_ORCH, AgentState.CRASHED -> Mono.just(WaitResponse)
```

Comment:
Can agent really send a `CRASHED` state? It should be a state that means that agent is offline. We shouldn't treat it silently, at least log a warning or an error.

Reply:
Agree, changed